src/diff/children.js

```diff
@@ -69,38 +69,30 @@ export function diffChildren(
 		// or we are rendering a component (e.g. setState) copy the oldVNodes so it can have
 		// it's own DOM & etc. pointers
 		else if (typeof childVNode == 'string' || typeof childVNode == 'number') {
-			childVNode = newParentVNode._children[i] = createVNode(
```

Comment:
Something's wrong here making the tests fail

Reply:
It looks like we're sometimes passing `VNodes` and sometimes backing nodes here. Need to investigate further.

Reply:
I must've goofed up somewhere